packages/plugin-ext/src/main/browser/webview/webview.ts

```diff
@@ -376,14 +385,7 @@ export class WebviewWidget extends BaseWidget implements StatefulWidget {
 
     protected onActivateRequest(msg: Message): void {
         super.onActivateRequest(msg);
-        this.focus();
```

Comment:
it is bogus, `onActivateRequest` is responsible to focus something, please revert

Reply:
Instead calling `this.focus()` I'm calling `this.node.focus()`, and then the focus message is sent inside focus event listener.
Do you see any issue with this?

Reply:
sorry overlooked it, let me try